Zilon.Core/Zilon.Core.Tests/Tactics/Behaviour/MoveIntentionTests.cs

```diff
@@ -133,10 +134,10 @@ public void CreateActorTask_CurrentMoveTaskWithOtherTarget_ReturnsNewMoveTask()
             var mapMock = new Mock<IMap>();
             var map = mapMock.Object;
 
-            var nodeMock = new Mock<IMapNode>();
+            var nodeMock = new Mock<IGraphNode>();
             var node = nodeMock.Object;
 
-            var node2Mock = new Mock<IMapNode>();
+            var node2Mock = new Mock<IGraphNode>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'node2Mock'.](https://app.codacy.com/app/kreghek/Zilon_Roguelike/pullRequest?prid=4592476)